scripts/install-tekton-pipelines.sh

```diff
@@ -4,6 +4,8 @@ set -eu
 KUBE_CONTEXT="--context kind-kind"
 KUBECTL_BIN="kubectl $KUBE_CONTEXT"
 TKN_VERSION="v0.22.0"
+TKN_DASHBOARD_VERSION="v0.17.0"
```

Comment:
@michaelsauter I think the Tekton dashboard is a nice to have when running against KinD.

Reply:
Why not, good idea. We might want to hide this behind a flag though so that we can disable in Github to save resources.